src/s3ql/fs.py

```diff
@@ -57,6 +57,17 @@
 else:
     ACL_ERRNO = errno.EOPNOTSUPP
 
+class _UpdateSize:
```

Comment:
We should just make this class writeable in pyfuse3 - this would also be useful for unit tests. But I think it's okay to work around this here.